app/tests/spec/lib/require-on-demand.js

```diff
@@ -34,26 +36,22 @@ define(function (require, exports, module) {
         });
       });
 
-      afterEach(function () {
-        sandbox.restore();
```

Comment:
I don't believe so since windowMock is created again for each test. The stubs used to be created by overwriting the real `window.require`, and I wanted to restore that object to its original state.